committer-tools/reviewers.py

```diff
@@ -87,9 +113,12 @@ def prompt_for_user():
             continue
 
     if selected_reviewers:
-        out = "\n\nReviewers: "
-        out += ", ".join([f"{name} <{email}>" for name, email, _ in selected_reviewers])
-        out += "\n"
-        print(out)
-
+        reviewer_message = f'\n\nReviewers: {", ".join([f"{name} <{email}>" for name, email, _ in selected_reviewers])}\n'
+        print(reviewer_message)
 
+        try:
+            pr_number = input("\nEnter the Pull Request number to append reviewer message to body (Ctrl+D or Ctrl+C to skip): ")
```

Comment:
This prompt is a bit long. How about "Pull Request (Ctrl+D or Ctrl+C to skip):" ?